addon/components/flexberry-map.js

```diff
@@ -77,6 +90,7 @@ export default Ember.Component.extend(
       this.set('_layer', map);
       this._addObservers();
       this._addEventListeners();
+      this.sendAction('leafletMapDidInit', map);
```

Comment:
Component called "flexberry-map".
Don't you think that {{flexberry-map leafletMapDidInit=(action "onLeafletMapDidInit")}} looks uglier then something like that {{flexberry-map init=(action "onMapInit")}}.
I mean that action name is too long & ugly.